docs/products/opensearch/reference/sunsetting-backward-compatible-aiven-for-opensearch.rst

```diff
@@ -0,0 +1,163 @@
+Sunsetting backward compatibility with Aiven for Elasticsearch
+==============================================================
+
+In September 2021, Aiven `introduced Aiven for OpenSearch® <https://aiven.io/blog/announcing-aiven-for-opensearch>`_ service that is backward compatible with Aiven for Elasticsearch.
+
+On 23 August 2022, a mandatory update will be scheduled to discontinue the backward compatibility with Aiven for Elasticsearch. In this article, we will explain what are the changes after performing the update.
+
+Aiven REST API
+--------------
+This section covers changes related to the REST API when the backward compatibility with Aiven for Elasticsearch is discontinued.
+
+Aiven for OpenSearch® ACL API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After 23 August 2022, we recommend you to switch to use `Aiven for OpenSearch® API <https://api.aiven.io/doc/#tag/Service:_OpenSearch>`_ instead of Aiven for Elasticsearch API. This includes using:
+
+* GET ``https://api.aiven.io/v1/project/{project}/service/{service_name}/opensearch/acl``
+* POST ``https://api.aiven.io/v1/project/{project}/service/{service_name}/opensearch/acl``
+* PUT ``https://api.aiven.io/v1/project/{project}/service/{service_name}/opensearch/acl``
+
+In favor of:
+
+* GET ``https://api.aiven.io/v1/project/{project}/service/{service_name}/elasticsearch/acl``
+* POST ``https://api.aiven.io/v1/project/{project}/service/{service_name}/elasticsearch/acl``
+* PUT ``https://api.aiven.io/v1/project/{project}/service/{service_name}/elasticsearch/acl``
+
+  
+.. seealso::
+
+	You can check the usage of Aiven API for Aiven for OpenSearch `on our official docs <https://api.aiven.io/doc/#tag/Service:_OpenSearch>`_.
+
+Aiven API to access service information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sunsetting the backwards compatibility with Aiven for Elasticsearch will bring changes in how to access service information using the Aiven API. You can find below the changes when you make a request to ``https://api.aiven.io/v1/project/{project}/service/{service_name}``:
```

Comment:
To align with other resources we need to explain what are {service_name}, {project}. For example, we have code samples here with variables. can be done in a similar way https://developer.aiven.io/docs/products/kafka/howto/connect-with-python.html